compose/ui/ui-test-junit4/src/desktopTest/kotlin/androidx/compose/ui/test/BasicTestTest.kt

```diff
@@ -137,4 +140,45 @@ class BasicTestTest {
             }
         }
     }
+
+    @Test
+    fun testIdlingResource() {
```

Comment:
Could this test live in "skikoTest" with `@IgnoreJsTarget` to run it for k/native too?
In skikoTests, instead of the rule there is a method "runSkikoComposeUiTest" in the scope of which `setContent` and `registerIdlingResource ` should be available?
If more tests from in this file can be in "skikoTests", then maybe it's better to do it in a separate PR.

Reply:
Right now I'm working on replicating Android's test API (which uses `rule`), so I prefer to test with that. Later on, we can work on commonizing the test API, and we'll probably move/copy some tests into common (without `rule`).